src/app/footer/versions.ts

```diff
@@ -0,0 +1,2 @@
+// this file is automatically generated by git.version.ts script
```

Comment:
- Don't commit generated files.
- Looks like travis needs to run "npm prebuild" for this file this file to be generated.